make/targets/openshift/operator/run-local.mk

```diff
@@ -0,0 +1,35 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+scripts_dir :=$(self_dir)../../../../scripts
+current_dir = $(shell pwd)
+
+define add-run-local-internal
```

Comment:
I don't think you need the macro, just define the target and variables

Reply:
Done